about.html

```diff
@@ -0,0 +1,173 @@
+      <nav class="nav-bar">
+        <div class="conference-logo-container">
+          <a href="./index.html"
+            ><img
+              class="conference-logo"
+              src="./asset/images/techLiveLogo.png"
+              alt="conference-logo"
+          /></a>
+        </div>
+        <div class="nav-links">
+          <ul class="nav-link-list">
+            <li class="nav-link logo-menu">
+              <a href="./index.html"
+                ><img
+                  src="./asset/images/techLiveLogo2.png"
+                  alt="TechLive Logo"
+              /></a>
+            </li>
+            <li class="nav-link"><a href="#about">About</a></li>
+            <li class="nav-link"><a href="#program">Program</a></li>
+            <li class="nav-link"><a href="#join">Join</a></li>
+            <li class="nav-link"><a href="#sponsor">Sponsor</a></li>
+            <li class="nav-link"><a href="#news">News</a></li>
+            <li class="nav-link active"><a href="#campaign">TL Campaign</a></li>
+          </ul>
+        </div>
+        <div class="hamburger-menu">
+          <div class="menu-icon"></div>
+        </div>
+      </nav>
+    </header>
+    <main class="main-section">
+      <section class="about-hero-section">
+        <div class="hero-container">
+          <p class="hero-heading-minor">Hello "tech" World!</p>
+          <h2 class="hero-heading-major">
+            TECHLive GLOBAL SUMMIT <br />
+            ABUJA, 2022
+          </h2>
+          <div class="hero-paragraph">
+            <p>
+              This year, 2022, the main issue of discussion is Software Error.
+              To Err is Human, To really Screw things up you need software and
+              as thus, we at Techlive bring together ideas, innovation, concepts
+              in handling software error from project conceptualization to
+              Project Deployment and maintenance.
+            </p>
+          </div>
+          <p class="hero-loc">
+            Please contact use Per Email, for any further questions about
+            TechLive Summit 2022!
+          </p>
+          <a class="hero-mail" href="http/email.com">techlive@istech.io</a>
+        </div>
+      </section>
+      <section class="about-logo-section">
+        <div class="about-logo-header">
+          <h2 class="header-text">
+            About TechLive <br />
+            Summit Logo
+          </h2>
+          <div class="header-line"></div>
+        </div>
+        <div class="logo-container">
+          <p class="logo-description">
+            The Logo of TechLive Summit 2022 was decided through the logo
+            competition from 28th July to 29th July
+          </p>
+          <div class="event-logo">
+            <img
+              class="event-logo-img"
+              src="./asset/images/techLiveLogo.png"
+              alt="event-logo"
+            />
+          </div>
+        </div>
+      </section>
+      <section class="past-summit-section">
```

Comment:
- [ ] Kindly check on the design of this section the images need to have opacity in the , check the comparison below 

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/181720330-4a946385-dfc9-40a3-ac02-6542c13a530a.png) |  ![image](https://user-images.githubusercontent.com/61408860/181720361-8f310d98-64c5-4351-8f9f-41df3a0cf784.png)

Reply:
![image](https://user-images.githubusercontent.com/95294594/181722097-51145b27-2a11-47aa-b4af-2643f0e079ed.png)
Kindly note that the effect is a hover effect and was implemented. Thanks for the comment